wiki/en/Installation-for-iOS.md

```diff
@@ -16,14 +16,26 @@ Make sure you've already read the [Getting Started](Getting-Started) page.
 
 If you have don't own a PC, we suggest you to buy a [Raspberry Pi](https://www.raspberrypi.org/){: target="_blank" rel="noopener noreferrer" } which is an inexpensive and small device that performs very well with Jamulus. iOS support is just a proof of concept and needs certain skills.
 
-* To install Jamulus on your iOS device, you need a computer running macOS. 
-* If you don't pay for an apple developer account, Jamulus will only run for at most 7 days before you need to re-install it.
+* To install Jamulus on your iOS device, you need a (free or paid) Apple Developer account and preferably a computer.
+* If you don't pay for an Apple developer account, Jamulus will only run for at most 7 days before you need to re-install it.
 * Jamulus on iOS works quite well on new devices if everything is set up correctly. However, we strongly recommend you use an audio interface and an USB-Ethernet adapter. To connect these to your iOS device, you should use something like the [Lightning to USB Camera Adapter](https://www.apple.com/uk/shop/product/MD821ZM/A/lightning-to-usb-camera-adapter) and a (powered) USB-hub.
 
 
 ## Install for iOS (for experienced users only)
+**Note**: We provide an unsigned .ipa file which you need to sign and then install on your device. If you are not Jailbroken (and have AppSync installed), you might need additional software to install Jamulus. There are multiple ways to do that, but this guide only explains the most standard approach. Please note that we assume you already have a (free) Apple developer account. If you don't have one, create an Apple ID and [Sign up for a developer account](https://developer.apple.com/membercenter). We recommend to create one without two factor authentification and unrelated to your potential personal account. You will need to enter the password in non-Apple software.
+
+1. [Download Jamulus (.ipa) on a PC/Mac]({{ site.download_root_link }}{{ site.download_file_names.ios }}){:.button}
+2. Install a program to sideload .ipa files ([Sideloadly](https://sideloadly.io/), [AltStore](https://altstore.io/), [Cydia Impactor](http://www.cydiaimpactor.com/) are possible options). This guide will focus on Sideloadly, but other software should work similar (Search for "Sideload .ipa with [name]" if you use different software).
```

Comment:
@ann0see I'm really against encouraging side-loading on iOS. While this scenario is possible, I'm not sure we should be 'recommending' it

Reply:
I see it as the only "unlimited" way to provide Jamulus. It *is* not perfect and inconvenient (7 days re-signing necessity). However we are independent from Apple and an unlimited number of users can install it.

TestFlight is an option but as far as I know limited? Also I think it’s more work on our side?
What about adding a disclaimer, shortening the instructions and in future work on publishing the App to TestFlight?

Reply:
I installed sonobus via TestFlight and they hit the maximum user count? Maybe provide both options but stress TestFlight?

Proper builds for the AppStore would be great – but we first need to get the UI fixed. Or do you think Apple would approve it as is?

Reply:
Ok. Removed specific recommendations and made the process a bit more general. I hope that’s ok for you.